_posts/2023-06-12-grpc-subsystem.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title:  "grpc Subsystem in WildFly"
-date:   2023-05-22
+title:  "grpc and WildFly - Part Ifootnote:[Part II: https://resteasy.dev/2023/06/11/grpc-in-wildfly-pt2/]"
```

Comment:
This part doesn't look right. I don't see how this can be parsed correctly. If anything, which I still don't think works, it should look more like:
```
title:  "grpc and WildFly - Part [2]I"
footnote:"[Part II: https://resteasy.dev/2023/06/11/grpc-in-wildfly-pt2/]"
```

Reply:
Ok, it's not worth spending time on. I'll put the footnote somewhere in the body. Thanks for looking at it.